packages/experimental/Tooltip/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@fluentui-react-native/tooltip",
+  "version": "0.1.0",
+  "description": "A cross-platform Tooltip component for React Native.",
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "typings": "lib/index.d.ts",
+  "onPublish": {
+    "main": "lib-commonjs/index.js",
+    "module": "lib/index.js"
+  },
+  "scripts": {
+    "build": "fluentui-scripts build",
+    "clean": "fluentui-scripts clean",
+    "depcheck": "fluentui-scripts depcheck",
+    "just": "fluentui-scripts",
+    "lint": "fluentui-scripts eslint",
+    "test": "fluentui-scripts jest",
+    "update-snapshots": "fluentui-scripts jest -u",
+    "prettier": "fluentui-scripts prettier",
+    "prettier-fix": "fluentui-scripts prettier --fix true"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui-react-native.git",
+    "directory": "packages/experimental/Tooltip"
+  },
+  "dependencies": {
+    "@fluentui-react-native/framework": "0.12.0",
+    "tslib": "^2.3.1"
+  },
+  "devDependencies": {
+    "@fluentui-react-native/component-cache": "^1.6.0",
+    "@fluentui-react-native/eslint-config-rules": "^0.1.1",
+    "@fluentui-react-native/experimental-button": "^0.19.0",
```

Comment:
experimental-button is a placeholder package to not break users, we should use button instead since that's where the really code lives.